app/controllers/shared/api.php

```diff
@@ -1,30 +1,24 @@
 <?php
 
 use Appwrite\Auth\Auth;
+use Appwrite\Event\Audit;
+use Appwrite\Event\Delete;
 use Appwrite\Event\Event;
+use Appwrite\Event\Mail;
 use Appwrite\Messaging\Adapter\Realtime;
+use Appwrite\Stats\Stats;
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Request;
 use Utopia\App;
 use Appwrite\Extend\Exception;
 use Utopia\Abuse\Abuse;
 use Utopia\Abuse\Adapters\TimeLimit;
+use Utopia\Database\Database;
 use Utopia\Database\Document;
 use Utopia\Database\Validator\Authorization;
+use Utopia\Registry\Registry;
 
-App::init(function ($utopia, $request, $response, $project, $user, $events, $audits, $mails, $usage, $deletes, $database, $dbForProject, $mode) {
-    /** @var Utopia\App $utopia */
-    /** @var Appwrite\Utopia\Request $request */
-    /** @var Appwrite\Utopia\Response $response */
-    /** @var Utopia\Database\Document $project */
-    /** @var Utopia\Database\Document $user */
-    /** @var Utopia\Registry\Registry $register */
-    /** @var Appwrite\Event\Event $events */
-    /** @var Appwrite\Event\Audit $audits */
-    /** @var Appwrite\Event\Mail $mails */
-    /** @var Appwrite\Stats\Stats $usage */
-    /** @var Appwrite\Event\Event $deletes */
-    /** @var Appwrite\Event\Event $database */
-    /** @var Appwrite\Event\Event $functions */
-    /** @var Utopia\Database\Database $dbForProject */
+App::init(function (App $utopia, Request $request, Response $response, Document $project, Document $user, Event $events, Audit $audits, Mail $mails, Stats $usage, Delete $deletes, Event $database, Database $dbForProject, bool $mode) {
```

Comment:
```suggestion
App::init(function (App $utopia, Request $request, Response $response, Document $project, Document $user, Event $events, Audit $audits, Mail $mails, Stats $usage, Delete $deletes, Event $database, Database $dbForProject, string $mode) {
```

Reply:
Mode is a string as seen here
https://github.com/appwrite/appwrite/blob/master/app/init.php#L71-L72